rules/prefer-key-over-key-code.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const keys = ['keyCode', 'charCode', 'which'];
+const keyCodeToKey = {
+	8: 'Backspace',
+	9: 'Tab',
+	13: 'Enter',
+	16: 'Shift',
+	17: 'Control',
+	18: 'Alt',
+	20: 'CapsLock',
+	27: 'Escape',
+	32: ' ',
+	33: 'PageUp',
+	34: 'PageDown',
+	35: 'End',
+	36: 'Home',
+	37: 'ArrowLeft',
+	38: 'ArrowUp',
+	39: 'ArrowDown',
+	40: 'ArrowDown',
+	46: 'Delete',
+	48: 0,
+	49: 1,
+	50: 2,
+	51: 3,
+	52: 4,
+	53: 5,
+	54: 6,
+	55: 7,
+	56: 8,
+	57: 9,
+	65: 'a',
+	66: 'b',
+	67: 'c',
+	68: 'd',
+	69: 'e',
+	70: 'f',
+	71: 'g',
+	72: 'h',
+	73: 'i',
+	74: 'j',
+	75: 'k',
+	76: 'l',
+	77: 'm',
+	78: 'n',
+	79: '0',
+	80: 'p',
+	81: 'q',
+	82: 'r',
+	83: 's',
+	84: 't',
+	85: 'u',
+	86: 'v',
+	87: 'w',
+	88: 'x',
+	89: 'y',
+	90: 'z'
```

Comment:
You don't need a table for common printable characters, you can instead use `String.fromCharCode`. Let's learn from React's code for `key`, which is almost exactly what we need (except we don't need the `normalizeKey` thingy) https://github.com/facebook/react/blob/b87aabd/packages/react-dom/src/events/getEventKey.js